frontend/public/locales/zh/machine-config-pools.json

```diff
@@ -5,8 +5,8 @@
   "Current configuration": "当前配置",
   "Current configuration source": "当前配置源",
   "Paused": "暂停",
-  "True": "True",
-  "False": "False",
+  "True": "真",
+  "False": "假",
```

Comment:
There are some other "True/False" on the machineconfigpool list page, not sure if all of them should be translated:
![Screenshot from 2021-02-19 17-28-23](https://user-images.githubusercontent.com/7313189/108487490-4ae6aa80-72da-11eb-9aed-dc87a5b2ff10.png)

Reply:
I think we probably need to translate true/false for conditions. We're showing the status value as-is currently.